contributed_definitions/nyaml/NXpeak.yaml

```diff
@@ -1,136 +1,42 @@
 category: base
 doc: |
-  Description of peaks, their functional form or measured support.
+  Base class for peaks, their functional form, and measured support.
 symbols:
   doc: |
     The symbols used in the schema to specify e.g. dimensions of arrays.
   n_support: |
     Number of support points
 type: group
 NXpeak(NXobject):
-  label:
+  label(NX_CHAR):
```

Comment:
No need to specify (NX_CHAR) here as it is still automatically inferred by nyaml2nxdl (to be compliant with NIAC).

Reply:
I know but I am still convinced for readability adding it explicitly is useful, thats why I decided to add it in all appdefs and base classes for em and apm, there is also a value of removing it to make the yaml file not look as wordy but then rightly so newcomers might not know immediately why in one case one has it and in another case not, i agree with you though it does not harm convenience issue